docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -341,6 +341,24 @@ spring:
 
 This will add `X-Request-Foo:Bar` header to the downstream request's headers for all matching requests.
 
+=== MapRequestHeader GatewayFilter Factory
+The MapRequestHeader GatewayFilter Factory takes a name and value parameter. It creates a new named header (config name) and the value is extracted out of some existing named header (config value) in the input. If the input header does not exist then the filter has no impact.
+
+.application.yml
+[source,yaml]
+----
+spring:
+  cloud:
+    gateway:
+      routes:
+      - id: add_request_header_route
+        uri: https://example.org
+        filters:
+        - MapRequestHeader=X-Request-Foo, Bar
+----
+
+This will add `X-Request-Foo:Bar` header to the downstream request's with the same value that the `Bar` header had.
```

Comment:
Shouldn't this be `X-Request-Foo` without the `:Bar`?

Reply:
@ryanjbaxter Indeed. I updated the documentation.